packaging/docker/build-images.sh

```diff
@@ -258,6 +259,7 @@ image_list=(
     'foundationdb-base'
     'foundationdb'
     'fdb-kubernetes-monitor'
+    'fdb-aws-s3-credentials-fetcher-sidecar'
```

Comment:
We probably have to do some additional changes in our release scripts, assuming we want to publish this script somewhere.

Reply:
Nod. This new image you mean? I can follow up w/ the release scripts after this goes in.